daemon/openrazer_daemon/dbus_services/dbus_methods/chroma_keyboard.py

```diff
@@ -78,11 +78,14 @@ def set_game_mode(self, enable):
 
     driver_path = self.get_driver_path('game_led_state')
 
-    for kb_int in self.additional_interfaces:
-        super_file = os.path.join(kb_int, 'key_super')
-        alt_tab = os.path.join(kb_int, 'key_alt_tab')
-        alt_f4 = os.path.join(kb_int, 'key_alt_f4')
-
+    # Fixes setting Game Mode/LED for Huntsman V2 w/ device
+    # files added to the usual MOUSE PROTOCOL Interface.
+    # Other devices seem to use the KEYBOARD PROTOCOL interface.
```

Comment:
This comment doesn't make much sense in the file, or reword it to something like "Check the usual MOUSE PROTOCOL interface first, and only then additional interfaces because ..."?

Also are you certain this won't break on other devices?

Reply:
I agree, updated to be more clear and less specific to a singular device [69c0db7](https://github.com/openrazer/openrazer/pull/1759/commits/69c0db7c1352e8351ca3208c387fb5cd54463906)